### DISCUSSION TITLE
dataset

### DISCUSSION THREAD

**Discussion by Winnie202** [open-mmlab/mmediting#1193] [Datasets]
What format should I define my anno.txt in, my data type is SRFolderMultipleGTDataset and the format is
data
    gt
     vsr1--1.jpg
                2.jpg...
     vsr2--1.jpg
               2.jpg...

**Comment by zengyh1900** [comment]
Hi @Winnie202 
You can check the introduction of dataset preparation here: https://mmediting.readthedocs.io/en/latest/_tmp/sr_datasets.html#reds-dataset 

Hope it helps!
